src/test/java/org/cloudfoundry/promregator/endpoint/SingleTargetMetricsEndpointTest.java

```diff
@@ -1,152 +0,0 @@
-package org.cloudfoundry.promregator.endpoint;
-
-import java.util.HashMap;
-import java.util.UUID;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-
-import org.cloudfoundry.promregator.JUnitTestUtils;
-import org.cloudfoundry.promregator.scanner.Instance;
-import org.cloudfoundry.promregator.scanner.ResolvedTarget;
-import org.cloudfoundry.promregator.textformat004.Parser;
-import org.junit.After;
-import org.junit.AfterClass;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.Mockito;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.http.ResponseEntity;
-import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.TestPropertySource;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-
-import io.prometheus.client.Collector.MetricFamilySamples;
-import io.prometheus.client.Collector.MetricFamilySamples.Sample;
-
-@RunWith(SpringJUnit4ClassRunner.class)
-@SpringBootTest(classes = MockedMetricsEndpointSpringApplication.class)
-@TestPropertySource(locations="default.properties")
-@ActiveProfiles("SingleTargetMetricsEndpointTest")
-public class SingleTargetMetricsEndpointTest {
```

Comment:
Note to myself: Yes, also this test may go, as we still have `SingleTargetMetricsEndpointTest.testGetMetrics`